README.md

```diff
@@ -336,6 +336,9 @@ TypeScript is a typed superset of JavaScript that compiles to plain JavaScript.
 * [Angular 2 Samples](https://github.com/thelgevold/angular-2-samples) Angular 2.0 sample components
 * [**{{** add_your_repo **}}**](https://github.com/angularclass/awesome-angular2/edit/gh-pages/README.md)
 
+#### TypeScript Demos, Samples, and Examples
```

Comment:
@r-park I think making a new section might be unnecessary as there already is one for Typescript [seed projects](https://github.com/AngularClass/awesome-angular2#typescript-seed-projects). Perhaps just add to it?